app/lib/methods/subscriptions/rooms.js

```diff
@@ -301,7 +303,10 @@ export default function subscribeRooms() {
 					const roomState = store.getState().room;
 					// Delete and remove events come from this stream
 					// Here we identify which one was triggered
-					if (data.rid === roomState.rid && roomState.isDeleting) {
+					if (sub._raw.team_main && data.rid === roomState.rid && roomState.isDeleting) {
```

Comment:
Can you unify these if conditions and call `store.dispatch(removedRoom());` only once?